assets/charts/components/headlamp/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for headlamp.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: joaquimrocha/headlamp
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: l8e
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
```

Comment:
If we do not use this value, should we keep it here?

Reply:
Good point.